web/models/task_list.ex

```diff
@@ -29,6 +31,10 @@ defmodule CodeCorps.TaskList do
         position: 4
       }
     ]
+
+    for task_list <- task_list_data do
+      TaskList.changeset(%TaskList{}, task_list)
+    end
   end
```

Comment:
Just a thought, but this might be nicely chained using

```Elixir
[%{name: "Index", position: 1}, ...]
|> Enum.map(fn (params) -> changeset(%TaskList{}, params) end)
```